### PR TITLE
NEW PARAMETER - navbarOffset

### PR DESCRIPTION
**Pull request by dlottich**
NEW PARAMETER - navbarOffset - defaults to 0, the navbar doesn't hide that many pixels from the bottom.

### CODE REVIEW

**Comment by vladimirnani** [comment]
Hey. Great idea doing same for hiding two navbars

**Comment by dlottich** [comment]
Can you help me get this to pass the Travis CI?

File: dist/jquery.bootstrap-autohidingnavbar.js [hunk 1]
```diff
@@ -6,6 +6,14 @@
  *  Made by István Ujj-Mészáros
  *  Under Apache License v2.0 License
  */
+/*
```
Comment (vladimirnani) [inline dist/jquery.bootstrap-autohidingnavbar.js]: @dlottich no need for extra comments :)

File: package.json [hunk 2]
```diff
@@ -6,27 +6,28 @@
     "name": "István Ujj-Mészáros",
     "url": "https://github.com/istvan-ujjmeszaros"
   },
-  "contributors": [
-  ],
+  "contributors": [],
```
Comment (vladimirnani) [inline package.json]: reverting` package.json` will fix ci

File: src/jquery.bootstrap-autohidingnavbar.js [hunk 3]
```diff
@@ -1,3 +1,11 @@
+/*
```
Comment (vladimirnani) [inline src/jquery.bootstrap-autohidingnavbar.js]: this comments also can be removed

**Comment by istvan-ujjmeszaros** [comment]
Thanks guys!

**Pull request merged**